src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -35,7 +35,29 @@ impl From<Vec<LazyBatchColumn>> for LazyBatchColumnVec {
     }
 }
 
+impl From<Vec<VectorValue>> for LazyBatchColumnVec {
+    #[inline]
+    fn from(columns: Vec<VectorValue>) -> Self {
+        LazyBatchColumnVec {
+            columns: columns
+                .into_iter()
+                .map(|v| LazyBatchColumn::from(v))
+                .collect(),
+        }
+    }
+}
+
 impl LazyBatchColumnVec {
+    /// Creates a new empty `LazyBatchColumnVec`, which does not have columns and rows.
+    ///
+    /// Because column numbers won't change, it means constructed instance will be always empty.
+    #[inline]
+    pub fn new_empty() -> Self {
+        Self {
+            columns: Vec::new(),
```

Comment:
Maybe implement `default` will be better.

Reply:
Maybe not a good idea, since `LazyBatchColumnVec`'s columns are immutable, which means when building an empty LazyBatchColumnVec, it will stay empty forever. This does not looks like to be a common case.